libs/ui/src/global_styles.tsx

```diff
@@ -1,16 +1,141 @@
+import 'normalize.css';
 import { createGlobalStyle } from 'styled-components';
 
+export interface GlobalStlyesProps {
+  isTouchscreen: boolean;
+  legacyBaseFontSizePx?: number;
+  legacyPrintFontSizePx?: number;
+}
+
 /**
  * Common global styling for VxSuite apps.
  *
- * TODO: Flesh out and vary based on selected pre-defined theme.
+ * TODO: Copied from old App.css files in the frontend packages - could probably
+ * use some cleanup and de-duping with the normalize.css styles we're already
+ * importing.
+ *
+ * TODO: Hardcode base64-encoded versions of our font files and reference here,
+ * so that everything's centralized and we don't have to have duplicate
+ * copies in each app's package.
  */
-export const GlobalStyles = createGlobalStyle`
+export const GlobalStyles = createGlobalStyle<GlobalStlyesProps>`
+  *,
+  *::before,
+  *::after {
+    box-sizing: inherit;
+  }
+
   html {
-    background: #edeff0;
-    color: #263238;
+    box-sizing: border-box;
+    background: ${(p) => p.theme.colors.background};
+    line-height: 1;
+    color: ${(p) => p.theme.colors.foreground};
     font-family: 'Vx Helvetica Neue', 'Noto Emoji', 'Helvetica Neue', sans-serif;
-    -webkit-font-smoothing: antialiased;
+    font-size: ${(p) => p.legacyBaseFontSizePx || p.theme.sizes.fontDefault}px;
+
     -moz-osx-font-smoothing: grayscale;
+    -webkit-font-smoothing: antialiased;
+
+    user-select: none;
+  }
+  @media print {
+    html {
+      background: #ffffff;
+
+      /* Adjust printed ballot font-size */
+      /* stylelint-disable-next-line declaration-no-important */
+      font-size: ${(p) => p.legacyPrintFontSizePx ?? 16}px !important;
+    }
+  }
```

Comment:
I wonder if this will break styles on our printing. I suggest you take a document from one of the apps and print (in Chrome) a copy from `main` and a copy from this branch. 

Printed documents are rendered outside of the app as an appended child to the document body, see `/libs/ui/src/print_element.tsx`. Previously, the `App.css` file would be applied everywhere in the body. But my understanding is that these styles will now only apply within the apps. 

If that's the case, I think that's good, but we'll have to move some of these print related styles into the `PrintOnly` styled component wrapping in `libs/ui/src/print_element.tsx`. That is at least one way of doing it. Passing in the legacy base font sizes, I'm not sure how best to do that. It will be outside of the `ThemeProvider` so we can't snatch it from the theme. Ideas?

And, if I'm misunderstanding this, and these styles still apply to the printed stuff, great.

Reply:
Yeah, it's a good nudge - the `createGlobalStyles` feature hoists all the global styles to the `<head>` section of the html file, so it should apply to everything in the document, in theory, including the print content.

I'll verify with a few test prints, just in case - forgot to test that path!

Love that we already have the `PrintOnly` component and the other print helpers! Was contemplating adding something like that when I noticed the print only styles in `App.css` - will see if I can move stuff over to use the component while I work through the design refresh.